frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { K8sKind, k8sList, referenceForModel } from '../../module/k8s';
+import { ResourceLink, resourcePathFromModel } from '../utils/resource-link';
+
+const globalConfigListStateToProps = ({k8s}) => ({
+  configResources: k8s.getIn(['RESOURCES', 'configResources']),
+});
+
+const ItemRow = ({item}) => {
+  return <div className="row co-resource-list__item">
+    <div className="col-sm-5 col-xs-6">
+      <ResourceLink kind={referenceForModel(item.model)} name={item.metadata.name} />
+    </div>
+    <div className="col-sm-5 col-xs-6">
+      {item.kind}
+    </div>
+    <div className="col-sm-2 hidden-xs">
+      <Link to={`${resourcePathFromModel(item.model)}/${item.metadata.name}/yaml`} className="btn btn-default pull-right">Edit YAML</Link>
+    </div>
+  </div>;
+};
+
+class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps> {
+  state = {
+    errorMessage: '',
+    items: [],
+  }
+
+  componentDidMount() {
+    Promise.all(this.props.configResources.map(model => {
+      return k8sList(model)
+        .catch(err => {
+          const errorMessage = err.message;
+          this.setState({ errorMessage });
+          return [];
+        }).then(items => items.map(i => ({...i, model})));
+    })).then((responses) => {
+      const items = _.sortBy(_.flatten(responses), 'metadata.name');
```

Comment:
I'd use `kind` as a secondary sort

```suggestion
      const items = _.orderBy(_.flatten(responses), ['metadata.name', 'kind'], ['asc', 'asc']);
```